support/cas-server-support-themes-collection/src/main/resources/static/themes/twbs/js/cas.js

```diff
@@ -0,0 +1,121 @@
+/* global trackGeoLocation, jqueryReady */
```

Comment:
Yes, there are a number of changes to support bootstrap rather than material design, as well as switching it to font-awesome icons rather than material icons.